eng/scripts/Build-Local.ps1

```diff
@@ -11,6 +11,9 @@ param(
     [switch] $VerifyNpx
 )
 
+# Add C:\Program Files\nodejs\npx to path
```

Comment:
This script can be run cross platform and this ties it to Windows.

Reply:
My mistake, this was a temporary line I add to get going.  Removed.